igorpro.css

```diff
@@ -0,0 +1,29 @@
+.highlight table td { padding: 5px; }
```

Comment:
Please remove this file - the theme is enough to generate it, and we don't store manifests of other themes.